packages/ckeditor5-html-support/tests/htmlcomment-integration.js

```diff
@@ -0,0 +1,1455 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* globals Event */
+
+import ClassicTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/classictesteditor';
+import Essentials from '@ckeditor/ckeditor5-essentials/src/essentials';
+import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
+
+import BlockQuoteEditing from '@ckeditor/ckeditor5-block-quote/src/blockquoteediting';
+
+import CodeBlockEditing from '@ckeditor/ckeditor5-code-block/src/codeblockediting';
+
+import HeadingEditing from '@ckeditor/ckeditor5-heading/src/headingediting';
+
+import HighlightEditing from '@ckeditor/ckeditor5-highlight/src/highlightediting';
+
+import HtmlEmbedEditing from '@ckeditor/ckeditor5-html-embed/src/htmlembedediting';
+
+import ImageBlockEditing from '@ckeditor/ckeditor5-image/src/image/imageblockediting';
+import ImageInlineEditing from '@ckeditor/ckeditor5-image/src/image/imageinlineediting';
+import ImageCaptionEditing from '@ckeditor/ckeditor5-image/src/imagecaption/imagecaptionediting';
+
+import IndentBlock from '@ckeditor/ckeditor5-indent/src/indentblock';
+import IndentEditing from '@ckeditor/ckeditor5-indent/src/indentediting';
+
+import LinkEditing from '@ckeditor/ckeditor5-link/src/linkediting';
+import LinkImageEditing from '@ckeditor/ckeditor5-link/src/linkimageediting';
+
+import ListEditing from '@ckeditor/ckeditor5-list/src/listediting';
+import ListStyleEditing from '@ckeditor/ckeditor5-list/src/liststyleediting';
+import TodoListEditing from '@ckeditor/ckeditor5-list/src/todolistediting';
+
+import MediaEmbedEditing from '@ckeditor/ckeditor5-media-embed/src/mediaembedediting';
+
+import SourceEditing from '@ckeditor/ckeditor5-source-editing/src/sourceediting';
+
+import TableEditing from '@ckeditor/ckeditor5-table/src/tableediting';
+import TableCaption from '@ckeditor/ckeditor5-table/src/tablecaption';
+
+import HtmlComment from '../src/htmlcomment';
+
+describe( 'HtmlComment integration', () => {
+	describe( 'integration with BlockQuote', () => {
+		let editor;
+
+		function getEditor( initialData = '' ) {
```

Comment:
Can we change all `getEditor` to `createEditor`?

Reply:
Done.